--- build.sh	2021-08-11 11:58:33.516632946 +0200
+++ build_new.sh	2021-08-12 14:50:43.601730625 +0200
@@ -28,7 +28,7 @@
 BUILD_SYMBOOGLIX=1
 BUILD_LOCKPWN=1
 BUILD_SMACK=1
-TEST_SMACK=1
+TEST_SMACK=0
 BUILD_LLVM=0 # LLVM is typically installed from packages (see below)
 BUILD_MONO=0 # mono is typically installed from packages (see below)
 
@@ -191,7 +191,7 @@
   DEPENDENCIES+=" clang-${LLVM_SHORT_VERSION} llvm-${LLVM_SHORT_VERSION} mono-complete libz-dev libedit-dev"
   ;;
 
-linux-@(ubuntu|neon)-16*)
+linux-@(ubuntu|neon)-@(16|18|20)*)
   Z3_DOWNLOAD_LINK="https://github.com/Z3Prover/z3/releases/download/z3-${Z3_SHORT_VERSION}/z3-${Z3_FULL_VERSION}-x64-ubuntu-16.04.zip"
   DEPENDENCIES+=" clang-${LLVM_SHORT_VERSION} llvm-${LLVM_SHORT_VERSION} mono-complete libz-dev libedit-dev"
   ;;
@@ -251,13 +251,13 @@
     sudo zypper --non-interactive install ${DEPENDENCIES}
     ;;
 
-  linux-@(ubuntu|neon)-1[46]*)
+  linux-@(ubuntu|neon)-@(14|16|18|20)*)
     RELEASE_VERSION=$(get-platform-trim "$(lsb_release -r)" | awk -F: '{print $2;}')
     case "$RELEASE_VERSION" in
     14*)
       UBUNTU_CODENAME="trusty"
       ;;
-    16*)
+    @(16|18|20)*)
       UBUNTU_CODENAME="xenial"
       ;;
     *)
@@ -369,7 +369,7 @@
   echo ". /usr/share/GNUstep/Makefiles/GNUstep.sh" >> ${SMACKENV}
 
   puts "Installed Objective-C"
-fi 
+fi
 
 if [ ${INSTALL_RUST} -eq 1 ] ; then
   puts "Installing Rust"
@@ -379,7 +379,7 @@
   cd rust-nightly-x86_64-unknown-linux-gnu
   sudo ./install.sh
   cd ..
-  
+
   puts "Installed Rust"
 fi
 
